src/defaults/tabSpace.tsx

```diff
@@ -135,10 +135,7 @@ class TabSpace extends VDomRenderer<TabSpace.Model> implements ITabSpace {
         super.onUpdateRequest(msg);
     }
 
-    private _onActiveCellChange = (
-        _tracker: INotebookTracker,
-        cell: Cell | null
-    ) => {
+    private _onActiveCellChange = (_tracker: INotebookTracker, cell: Cell | null) => {
```

Comment:
Did the signature of these change?  I thought they converted to `(_tracker: INotebookTracker, cell: Cell) => void`. Also why is there the need to add all of these `!` overrides.